build.gradle

```diff
@@ -272,3 +274,10 @@ task run(type: Run) {
   group = 'Verification'
   impliesSubProjects = true
 }
+
+task buildSrcIntegTest(type: GradleBuild) {
```

Comment:
I'm concerned about startup time. This will make it slower to run a build from scratch.